function err = Exception(id, varargin)
    err = MException(id, varargin{:});
end
